docs/guides/how-do-i-use-orbstack.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Using Lando with Orbstack
+description: Learn how to use Lando with Orbstack today.
+guide: true
+
+authors:
+  - name: Oscar Arzola
+    pic: https://avatars.githubusercontent.com/u/4997549?v=4
+    link: https://twitter.com/init_sh
+updated:
+  timestamp: 1733175010
+---
+
+# Using Lando with Orbstack
+
+[Orbstack](https://docs.orbstack.dev/) is a lightweight and ⚡️ lightning-fast Docker provider that can serve as a drop-in replacement for Docker Desktop. It is an excellent alternative for users who want to use Lando without Docker Desktop.
+
+Official support for swapping Docker providers will be available in [Lando 4](https://lando.dev/blog/2023/01/23/roadmap-of-2023.html). In the meantime, you can use [Orbstack](https://orbstack.dev/) as a drop-in replacement.
+
+After installing Lando, follow these steps:
+
+1. Kill all running containers, close Docker Desktop and ensure that auto-start is disabled in Docker Desktop settings.
+2. **Install Orbstack** (You can migrate your existing containers easily; see below).
+3. Lando only checks for the presence of the **Docker.app** client; **it does not require Docker Desktop to be running**. By opening Orbstack, it binds the necessary Docker CLI commands that Lando expects and uses. This allows you to run Lando seamlessly without relying on Docker Desktop.
+
+## Installing Orbstack
```

Comment:
Since these sections are small, I think it would flow better to just include these in the steps that you lay out above

Reply:
Perhaps give some common install instructions here so that most users don't have to navigate away.